plugin.video.101barz/addon.xml

```diff
@@ -23,13 +25,13 @@
         <language>nl</language>
         <platform>all</platform>
         <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
GPL-2.0-only or GPL-2.0-or-later if possible

Reply:
Fixed it.